aixplain/modules/pipeline/designer/base.py

```diff
@@ -280,14 +281,31 @@ def __getitem__(self, code: str) -> Param:
                 return param
         raise KeyError(f"Parameter with code '{code}' not found.")
 
+    def special_prompt_handling(self, code: str, value: str) -> None:
```

Comment:
Should you add a unit test for this function just in case?

Reply:
added